frontend/src/components/studies/StudyEditForm.tsx

```diff
@@ -114,6 +121,16 @@ const StudyEditForm = ({ workflows, isCreatingStudy, onSubmit, formErrors }: Cre
 					onChange={workflowWasSelected}
 				/>
 			</Item>
+			<Item
+				name="description"
+				label="Description"
+				{...itemValidation('description')}
+			>
+				<Input
+					defaultValue={''}
```

Comment:
null might be a better default since it would prevent to populate the DB with empty strings unless you do additional tests.

Reply:
Not possible, but I can make it None at the viewset or service.